build_debian.sh

```diff
@@ -318,6 +318,11 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     haveged                 \
     jq
 
+## Configure kdump-tools with additional flags for vmcore dump
+sudo LANG=C chroot $FILESYSTEM_ROOT /bin/bash -c 'cat >> /etc/default/kdump-tools <<"EOF"
+KDUMP_SYSCTL="kernel.panic_on_oops=1 kernel.panic_on_unrecovered_nmi=1 kernel.panic_on_io_nmi=1 kernel.panic_on_stackoverflow=1 kernel.hung_task_panic=1 kernel.softlockup_panic=1 kernel.unknown_nmi_panic=1 vm.panic_on_oom=1"
```

Comment:
please check pr https://github.com/Azure/sonic-buildimage/pull/6113 this one does not seem to be align with the design.

Reply:
@lguohan  can you please check it now?

Reply:
@shlomibitton 

The below is part of the description of "KDUMP_SYSCTL"
    Each time a kdump kernel is loaded, “sysctl -w $KDUMP_SYSCTL” is also executed

In my opinion, we should not tie the possible causes of kernel panic with kdump being enabled. For example, certain users may not want to panic on oom or on soft lockup but still would want to panic and take kdump on other possibilities. So it is best to define these parameters in a file placed in /etc/sysctl.d/ or in /etc/sysctl.conf instead of tying them with the default kdump config.

Reply:
I agree, I have added a new file to apply these changes.